packages/babel-preset-default/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### New Features
+
+- The bundled `@babel/core` dependency has been updated from requiring `^7.11.6` to requiring `^7.12.9`. All other Babel plugins were updated to the latest version (see [Highlights](https://babeljs.io/blog/2020/10/15/7.12.0)).
+
+## 4.19.0 (2020-10-01)
```

Comment:
This is surprising, I will investigate further why it doesn't have the heading with the section, I would expect here a major version because of breaking changes listed.

Reply:
I inferred the missing changelog entry from `npm info`, including the approximate release date :)

```
$ npm info "@wordpress/babel-preset-default"

@wordpress/babel-preset-default@4.19.0 | GPL-2.0-or-later | deps: 10 | versions: 46
Default Babel preset for WordPress development.
https://github.com/WordPress/gutenberg/tree/master/packages/babel-preset-default/README.md

dependencies:
@babel/core: ^7.11.6                              @babel/runtime: ^7.11.2                           @wordpress/warning: ^1.3.0                        
@babel/plugin-transform-react-jsx: ^7.10.4        @wordpress/babel-plugin-import-jsx-pragma: ^2.7.0 core-js: ^3.6.4                                   
@babel/plugin-transform-runtime: ^7.11.5          @wordpress/browserslist-config: ^2.7.0            
@babel/preset-env: ^7.11.5                        @wordpress/element: ^2.18.0                       

dist-tags:
latest: 4.19.0     next: 4.15.1-rc.0  

published 2 months ago by wpisabel <isabel.brison@automattic.com>
```

The Babel upgrade is there, and the `shippedProposals` removal is there, too.

Reply:
I think the issue is a clear example that CHANGELOG parsing is too naive. The issue was that there was a wrong heading level ...

Related PR: https://github.com/WordPress/gutenberg/pull/22083
Follow-up release commit: fdeb10aca77868a9f7d0400c652e7e1709f5be44

I updated the CHANGELOG with d1a52f701dce40dadce89d74eaa73b438fbd8d45.

It was expected to be a major update but it was a minor instead 😅